libbeat/docs/upgrading.asciidoc

```diff
@@ -114,12 +137,25 @@ have been renamed. See {beats-ref}/breaking-changes-7.0.html[Breaking
 changes in 7.0] for the full list of changed names.
 
 To help you transition to the new fields, we provide a compatibility layer in
-the form of ECS-compatible field aliases. To use the aliases, set
-`migration.6_to_7.enabled: true` in the Beat's configuration file *before* you
-upgrade the {es} index template to 7.0.
+the form of ECS-compatible field aliases. To use the aliases, set the following
+option in the Beat's configuration file *before* you upgrade the {es} index
+template to 7.0.
+
+[source,yaml]
+----
+migration.6_to_7.enabled: true
+----
+
+The field aliases enable you to use 6.x dashboards and visualizations
+with indices created by {beats} 7.0 or later. The aliases do *not* work
+with saved searches or with API calls that manipulate documents directly.
 
-The compatibility layer enables you to use 6.x dashboards and visualizations
-with indices created by {beats} 7.0 or later.
+We strongly advise that you adjust your custom {kib} dashboards, machine
+learning jobs, and other content to use the new ECS field names. The aliases
+may be removed in version 8.0.
+
+// REVIEWERS: Is the 8.0 statement correct? Should I mention it here? If so,
+// should I say "will be removed"?
```

Comment:
@ruflin for how long do we plan to maintain this compatibility layer? 

How about just saying: `will be removed in future releases` ?

Reply:
It should stick around for the 7.x lifecycle but removed in 8.0

Reply:
Agree with Ruflin on the lifetime of the setting.

In the migrating Beats article, I'm suggesting people can disable the setting after they've performed their upgrade, gotten rid of all of their use of old fields. This way, next minor upgrade will no longer create them.

May be worth mentioning here as well, but not a deal breaker.